### ISSUE TITLE
Add new group and user for flink

### ISSUE THREAD

**Issue by hongzhanchen** [clearlinux/distribution#1178]
I recently tried to create docker image clearlinux/flink and made it align with official flink one.  The official docker file https://github.com/docker-flink/docker-flink/blob/29f5bebe0261f3103a49a5cab5de982d4d16fb76/1.8/scala_2.11-debian/Dockerfile will create new user and group flink with directly calling useradd and groupadd which is not allowed in our CL.  
It seems that  I need to create patch for package clr-systemd-cofig to modify sysusers.d/clear.conf but I do not have authority to access source and  can not create new tar file for clr-systemd-config. Could anyone help create it?

**Comment by miguelinux** [comment]
I read [/opt/flink](https://github.com/docker-flink/docker-flink/blob/29f5bebe0261f3103a49a5cab5de982d4d16fb76/1.8/scala_2.11-debian/Dockerfile#L52) as $HOME,

Does the **flink** user needs a **$HOME** or it could be any other directory?
Does the $HOME is used to write some configs?

Usually systemd users are for daemons and its $HOME is /.

You can use **useradd** and **groupadd** on Clear Linux as it shown in the Dockerfile.

**Comment by hongzhanchen** [comment]
@miguelinux ,Actually ,I did same in https://github.com/clearlinux/dockerfiles/pull/233 as you mentioned  and just used useradd and groupadd in the Docker file. But gtkramer  suggested I may use different way. And then I searched commit history and found that CL add system user like httpd and kafka and zookeeper in sysusers.d/clear.conf . There is also similiar issue https://github.com/clearlinux/distribution/issues/960.
